config/crd/bases/ovn.openstack.org_ovndbclusters.yaml

```diff
@@ -52,6 +52,12 @@ spec:
                 description: DBType - NB or SB
                 pattern: ^NB|SB$
                 type: string
+              electionTimer:
+                default: 60000
```

Comment:
this looks much, isn't 10sec good enough?

Reply:
Yeah, another bad copy-paste :)